docs/topics/intellij-idea-plugin-reviewing-issues.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * docs/intellij-idea-plugin-guide/master.adoc
+
+[id="intellij-idea-plugin-reviewing-issues_{context}"]
+= Reviewing issues
+
+You can use the {ProductShortName} plugin icons to prioritize issues based on their severity. You can  see which issues have a _Quick Fix_ automatic code replacement and which do not.
```

Comment:
Seems like _Quick Fix_ means "automatic code replacement". If so, then define it here, and just use "Quick Fix" subsequently, without "automatic code replacement".

Reply:
Done.